### PR TITLE
Added custom logger

### PR DESCRIPTION
**Pull request by Kronifer**
We'll have fancy logs now

### CODE REVIEW

**Review by AbooMinister25 — CHANGES_REQUESTED** [review]
Alright, the general layout seems fine, but why is it a context manager? I don't really see a need for it to be one, most other logging libraries work fine without being a context manager. Its not really a *bad* design decision, but it may end up being unnecessarily verbose for general logging purposes.

**Comment by AbooMinister25** [comment]
Also, this isn't strictly required, but it might be nice to add a `success` and `debug` logging options.

**Comment by Kronifer** [comment]
How do you mean

File: backend/core/logger.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+from datetime import datetime
+from sys import stdout
+
+COLORS = {
+    "end": "\033[0m",
+    "error": "\033[31m",
+    "info": "\033[32m",
+    "warning": "\033[93m",
+    "timestamp": "\033[96m",
+}
```
Comment (doublevcodes) [inline backend/core/logger.py:11]: I am not sure ANSI escapes work on some Windows command prompts. Would it be better to import a formatting library

File: backend/core/logger.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    f"{logtype} | {COLORS.get('timestamp')}{datetime.utcnow().strftime('%d-%m-%Y %H:%M:%S UTC')}{COLORS.get('end')} | {body}\n"
+                )
+            else:
+                with open(self.filename, "a+") as file:
+                    file.write(
+                        f"{logtype} | {datetime.utcnow().strftime('%d-%m-%Y %H:%M:%S UTC')} | {body}\n"
+                    )
+                stdout.write(
+                    f"{logtype} | {datetime.utcnow().strftime('%d-%m-%Y %H:%M:%S UTC')} | {body}\n"
+                )
```
Comment (doublevcodes) [inline backend/core/logger.py:89]: There's probably a way of condensing this I just can't think of it

**Comment by dawnofmidnight** [comment]
This is no longer needed as we're using [`loguru`](https://loguru.rtfd.io/).

**Pull request closed without merging**